src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -286,6 +294,26 @@ public async Task VaultTimeoutAsync(bool promptOptions = true, int? newTimeout =
             }
         }
 
+        public async Task AppCenterReportingAsync()
+        {
+#if !FDROID && !DEBUG
+            var options = _appcenterActions.Select(s => s == _appCenterReportsDisplayValue ? $"✓ {s}" : s).ToArray();
+            var selection = await Page.DisplayActionSheet(AppResources.ReportAppLogsDescription, AppResources.Cancel, null, options);
+
+            if (selection == null || selection == AppResources.Cancel)
+            {
+                return;
+            }
+
+            var cleanSelection = selection.Replace("✓ ", string.Empty);
+            var enableAppcenter = cleanSelection == AppResources.Yes;
+            await Microsoft.AppCenter.Crashes.Crashes.SetEnabledAsync(enableAppcenter);
```

Comment:
Even though I know it's used like this on other parts of the code, we can avoid the replace and just compare directly:
```c#
await Microsoft.AppCenter.Crashes.Crashes.SetEnabledAsync(CompareSelection(selection, AppResources.Yes));

// Directly
bool CompareSelection(string selection, string compareTo) => selection == $"✓ {compareTo}";

// Substring from 2nd index (avoiding the tick comparison)
bool CompareSelection(string selection, string compareTo) => selection?.Substring(2) == compareTo;
```